sdk/identity/azcertificatemanager/cloud_config.go

```diff
@@ -0,0 +1,42 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azcertificatemanager
+
+import "github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+
+// Cloud Service Names for Monitor Query Logs and Metrics, used to identify the respective cloud.ServiceConfiguration
+const (
+	ServiceNameLogs    cloud.ServiceName = "azcertificatemanagerLogs"
```

Comment:
The names of these constants are service-specific.  The pattern is `ServiceName<suffix>`.